.env

```diff
@@ -12,5 +14,14 @@ UPSTREAM_DNS=8.8.8.8
 ## Storage path for the cached data
 CACHE_ROOT=/lancache
```

Comment:
any reason we are using /lancache, when in all the tutorials we use /data/cache?

Reply:
This was part of the original setup, not altered in this changeset. Although I'd be +0 on changing this to `/data` - it's important to remember in this context that this is used as a base path for both the cache data and logfiles. The original value does make slightly more sense in that case, as at least then it's somewhat identifiable (if we were following the Linux Standard Base path rules, we'd use something like `/srv/lancache`).

Reply:
/srv/lancache or /lancache makes sense, we should update the standard helps later to match this if we make the change!

Reply:
Had a think about this... can we just do this as a relative path by default?

`CACHE_ROOT=lancache`

that way if they forget to change this it will at least be present in the same place not disappear into the filesystem?

Reply:
If we did change this, making it explicitly relative (`CACHE_ROOT=./lancache`) would probably be sensible.

Reply:
yup, I think we should do that. Makes for less broken/lost things if they forgot to edit it!

Reply:
Changed.